_includes/scenarios/integrate/create-own-identityattribute.md

```diff
@@ -0,0 +1,78 @@
+
+```jsonc
+{
+  "content": {
+    "@type": "IdentityAttribute",
+    "owner": "<Address of your Connector>",
+    "validFrom": "<start of Attribute validity>",
+    "validTo": "<end of Attribute validity>",
+    "value": {
+      // IdentityAttribute Value
+      ...
+    },
+    "tags": ["<additional information 1>", ..., "<additional information m>"]
+  }
+}
+```
+
+You need to replace the placeholders marked with `<...>` appropriately. Also, it is necessary that you insert one of the available [IdentityAttribute Values]({% link _docs_integrate/attribute-values.md %}#identity-attributes) into the `value` property. You are not allowed to specify the Address of a Connector other than your own as the value for the `owner` property, as the [Create an Attribute]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}) use case relates to the creation of IdentityAttributes for yourself and not for others. For that reason, it is also possible to omit the `owner` property. In that case, your Connector automatically becomes the owner of the IdentityAttribute. Note that the properties `validFrom`, `validTo` and `tags` are optional, so you can omit them as well.
+
+You can query your Connector's Address by proceeding as documented in the [Get currently used Identity]({% link _docs_use-cases/use-case-transport-get-currently-used-identity.md %}) use case.
+{: .notice--info}
+
+## Process
+
+As you can see from the diagram below, after you have entered the input values to create an IdentityAttribute for your own Connector, a few checks are performed to verify the correctness of your input. If the input is not entered correctly, an [error message]({% link _docs_integrate/error-codes.md %}) is sent in response. Otherwise, a check is performed whether the input values for the properties of the specified IdentityAttribute Value meet the validation criteria documented on the [IdentityAttribute Values]({% link _docs_integrate/attribute-values.md %}#identity-attributes) page. Assuming a successful validation, the IdentityAttribute to be created is saved as a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute). If it is a simple IdentityAttribute, a success response is sent directly. In the case of a complex IdentityAttribute, on the other hand, another LocalAttribute is created beforehand for each of its appropriate components. These LocalAttributes of the components contain the `id` of the LocalAttribute belonging to the complex IdentityAttribute within their `parentId` property.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/310cea0e-6f6f-4ee0-9efd-55e180ec5dda" id="WT4OFNWd3bcS"></iframe></div>
```

Comment:
I suppose this diagram needs to be adjusted to not have the separate chack for the owner. Also, for the first cross section I'd reformulate the text to "**Valid** input entered for creating an IdentityAttribute?". And writing about complex Attributes, I'd again use the term child Attribute.